compiler/src/parsing/parser.messages

```diff
@@ -4156,7 +4156,7 @@ program: LET LPAREN WASMI64 COMMA WHILE
 ## pattern COMMA
 ##
 
-Expected a pattern or an immediate `)` for a single-argument tuple pattern.
+Expected at least another pattern because single-argument tuple patterns are not supported.
```

Comment:
People aren't super familiar with single-argument tuples and most languages don't support them, so I don't think we need to mention them at all. This can actually just join the generic `Expected a pattern` message.